microsoft-edge/web-platform/password-reveal.md

```diff
@@ -0,0 +1,75 @@
+---
+description: This page provides guidance on controlling the display of the password reveal button
+title: Customizing the password reveal button
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/25/2020
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, compatibility, web platform, password reveal, eye icon
+---
+# Customizing the password reveal button  
+
+The `password` input type in Microsoft Edge includes a **password reveal** button.  This control enables a user to verify their password is entered correctly in the **password** field.  After a user has entered text in the **password** field, they may choose the **password reveal** button or select `Alt`+`F8` to toggle visibility of their input:  
+
+:::row:::
+   :::column span="":::
+      The password reveal button appears during text entry
+      
+      :::image type="complex" source="../media/web-platform/password-reveal/password-eye.png" alt-text="A **password** field with dots hiding the characters entered by a user.  An icon shaped like an eye is displayed within the right-hand side of the **password** field.  " lightbox="../media/web-platform/password-reveal/password-eye.png":::

```

Comment:
@jm-trd-ms is there another way we can present this information? The lightbox magnification icon ends up getting displayed over the password reveal icon, so it ends up obscuring the most important part of the image until you open the lightbox.

Reply:
Images have been updated.